parlai/core/torch_agent.py

```diff
@@ -429,8 +429,9 @@ def report(self):
         metrics = {}
         # heads up, if you have multiple optimizers, or different parameter
         # groups, this could be misleading
-        current_lr = round_sigfigs(self.optimizer.param_groups[0]['lr'], 4)
-        metrics['lr'] = round_sigfigs(current_lr, 4)
+        if hasattr(self, 'optimizer'):
```

Comment:
I ended up needing to hotfix this stuff in #1431, so go ahead and drop these lines.

Reply:
dropped